src/java/com/sarality/app/view/dialog/BaseDialogFragment.java

```diff
@@ -27,8 +17,8 @@
   @Override
   public Dialog onCreateDialog(Bundle savedInstanceState) {
     AlertDialog.Builder builder = new AlertDialog.Builder(getActivity());
-    getActionManager().setup(builder);
     configure(builder);
+    getActionManager().setup(builder);
```

Comment:
i am using the Action Manager. 
Action Manager -> setPositiveButtonAction . It creates the DialogButtonAction and saves it.
since the setup was already called, the action is never set on the builder and hence never on the dialog.